lib/PhantomEnvironment.js

```diff
@@ -157,151 +221,134 @@ class PhantomEnvironment extends AbstractEnvironment {
     return this._options[name];
   }
 
-  evaluateJs() {
-    const deferred = vow.defer(),
-      page = this._page,
-      args = Array.prototype.slice.call(arguments, 0);
+  evaluateJs(...args) {
```

Comment:
Would be very helpful if you can state here JSDoc with an example how to call this function

Reply:
it was not changed, but I agree jsdoc is needed. I added a point into https://github.com/redco/goose-phantom-environment/issues/1 since I think docs should be fixed in separate PR